modular_sand/code/game/objects/structures/fauna.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/kirbyplants/brass
+	name = "cog plant" //cog plant my beloved
+	desc = "Weird looking plant that has spinning gear, ressembles a sunflower?.."
+	icon = 'modular_nostra/icons/obj/flora/plants.dmi'
```

Comment:
```suggestion
	icon = 'modular_sand/icons/obj/flora/plants.dmi'
```
this don't seem to be a valid path

Reply:
oh thats because i literally took it from nostra